### PR TITLE
Fix request/response header validation

### PR DESCRIPTION
**Pull request by itaysk**
fix https://github.com/istio/istio/issues/10118
in place of https://github.com/istio/istio/pull/10119

cc: @ymesika

### CODE REVIEW

File: pilot/pkg/model/validation.go [hunk 1]
```diff
@@ -28,7 +28,6 @@ import (
 	"github.com/gogo/protobuf/proto"
 	"github.com/gogo/protobuf/types"
 	"github.com/hashicorp/go-multierror"
-
```
Comment (ymesika) [inline pilot/pkg/model/validation.go]: We usually separate the 3rd party from project's import with an empty line

Comment (itaysk) [reply pilot/pkg/model/validation.go]: that was one of my formatting plugins.. fixed

**Pull request merged**